pom.xml

```diff
@@ -33,6 +33,11 @@
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>branch-api</artifactId>
```

Comment:
:bug: for this dependency. If `scm-api` is missing some APIs we need for this feature, they should be added there, in a style consistent with the abstracted design of `SCMSourceOwners` etc., with any required implementations added to `branch-api`.